source/index.html

```diff
@@ -5,15 +5,23 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>HTML Academy: Кэт энерджи</title>
+  <link rel="icon" href="../favicon.ico">
+  <link rel="icon" href="../favicon.svg">
+  <link rel="apple-touch-icon" href="../apple-touch-icon.png" />
+  <link rel="manifest" href="../manifest.json" />
```

Comment:
```suggestion
  <link rel="icon" href="favicon.ico">
  <link rel="icon" href="favicon.svg">
  <link rel="apple-touch-icon" href="apple-touch-icon.png" />
  <link rel="manifest" href="manifest.json" />
```

и перенеси их в папку source, потому что это корень, а то где ты положила, это не корень сайта, и там лежат только служебные файлы